omnigibson/robots/stretch.py

```diff
@@ -0,0 +1,203 @@
+import os
+
+import numpy as np
+
+from omnigibson.macros import gm
+from omnigibson.robots.active_camera_robot import ActiveCameraRobot
+from omnigibson.robots.manipulation_robot import GraspingPoint, ManipulationRobot
+from omnigibson.robots.two_wheel_robot import TwoWheelRobot
+
+
+class Stretch(ManipulationRobot, TwoWheelRobot, ActiveCameraRobot):
```

Comment:
why is this in this PR?

Reply:
I believe that this comes from merging og-develop